CustomAlert/Classes/CustomAlert.swift

```diff
@@ -10,29 +10,42 @@ import UIKit
 
 public protocol CustomAlertDelegate: class {
     func dismissAlert(animated: Bool)
+    func dismissAlert(animated: Bool, completion: @escaping () -> Void)
```

Comment:
nechces to radsej spojit v jednu metodu ktora ma default value na ten completion?

Reply:
Neda se nastavit default value do protokolu, jedine muzu proste to volat s empty completion, ale nevim, ci je to lepsi :)

Reply:
ok